src/components/Combobox/Combobox.tsx

```diff
@@ -212,6 +212,7 @@ function Combobox<Item>({
                 <MenuItem
                   {...getItemProps({ item, disabled: disableItem(item) })}
                   as="li"
+                  data-testid={__DEV__ && slugify(label + '-' + itemToString(item))}
```

Comment:
can we add a ternary? Like so: `__DEV__ ? ... : undefined`

This way `data-testid` won't be present in production. Currently, it's `data-testid="false"`

Reply:
Done!